command/command.go

```diff
@@ -0,0 +1,22 @@
+package command
+
+import (
+	"os/exec"
+	"strings"
+)
+
+// RunCommand runs `command`, replacing the string "${url}" with `url`.
+func RunCommand(command string, url string) (string, error) {
+	cmdWithURL := strings.ReplaceAll(command, "${url}", url)
+	cmdSplit := strings.SplitN(cmdWithURL, " ", 2)
```

Comment:
Using a simple split like this will fail on commands with quoted strings. It might be better to use [go-shellquote](https://github.com/kballard/go-shellquote) or [something else](https://stackoverflow.com/questions/47489745/splitting-a-string-at-space-except-inside-quotation-marks).